internal/olm/operator/registry/index_image.go

```diff
@@ -59,20 +66,40 @@ const (
 	registryPodNameAnnotation = operatorFrameworkGroup + "/registry-pod-name"
 )
 
+const (
+	SchemaChannel  = "olm.channel"
+	SchemaPackage  = "olm.package"
+	DefaultChannel = "develop"
```

Comment:
I think I changed the default channel name to `operator-sdk-run` instead.

```suggestion
	DefaultChannel = "operator-sdk-run"
```